<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logindialog</class>
 <widget class="QDialog" name="logindialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>logindialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>380</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>380</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 249, 244);</string>
     </property>
     <widget class="Title" name="wtitle" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>32</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushbutton:hover{ 
	background-color: rgb(255, 0, 0);
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>291</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_date">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(90, 228, 255);
font: italic 28pt &quot;Comic Sans MS&quot;;
</string>
         </property>
         <property name="text">
          <string>date</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>291</width>
        <height>70</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,9,1">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8">
           <item>
            <widget class="QLineEdit" name="lineEdit_account">
             <property name="minimumSize">
              <size>
               <width>215</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>215</width>
               <height>20</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>输入用户名</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8">
           <item>
            <widget class="QLineEdit" name="lineEdit_password">
             <property name="minimumSize">
              <size>
               <width>215</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>215</width>
               <height>20</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>输入密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>291</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8,2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_login">
           <property name="minimumSize">
            <size>
             <width>176</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>176</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: bold 14px;
	border: 2px solid  rgb(255, 255, 255);
	border-color: rgb(255, 255, 255);
	background-color: rgb(90, 228, 255);    
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5ae4ff, stop: 1 #56dcf3);
  }

  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #56dcf3, stop: 1 #5ae4ff);
  }


</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_register">
           <property name="minimumSize">
            <size>
             <width>176</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>176</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: bold 14px;
	border-color: rgb(255, 255, 255);
	background-color: rgb(90, 228, 255);
    border: 2px solid  rgb(255, 255, 255);
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5ae4ff, stop: 1 #56dcf3);
  }

  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #56dcf3, stop: 1 #5ae4ff);
  }
 
  
</string>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Title</class>
   <extends>QWidget</extends>
   <header>title.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
